jest-common/src/main/java/io/searchbox/indices/script/DeleteIndexedScript.java

```diff
@@ -0,0 +1,64 @@
+package io.searchbox.indices.script;
+
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.GenericResultAbstractAction;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+import static io.searchbox.indices.script.ScriptLanguage.GROOVY;
+
+public class DeleteIndexedScript extends GenericResultAbstractAction {
+
+  private final String scriptName;
+  private final ScriptLanguage scriptLanguage;
+
+  public DeleteIndexedScript(Builder builder) {
+    super(builder);
+    this.scriptName = builder.scriptName;
+    this.scriptLanguage = builder.scriptLanguage;
+    setURI(buildURI());
+  }
+
+  protected String buildURI() {
+    try {
+      return super.buildURI() + "/_scripts/" + scriptLanguage.pathParameterName + "/" + URLEncoder.encode(scriptName, CHARSET);
+    } catch (UnsupportedEncodingException e) {
+      throw new RuntimeException(e);
```

Comment:
I wasn't sure, if its useful to URL encode the index name.
I opted for doing so, to be more safe.
Because of your comment, I'm still struggling.
Under the assumption, that users (hopefully) know that a script name should be choosen safely (a-z,0-9 for example), we could remove the encoding part.
This would be consistent with the other code.
The consequences are clear: it will still work for 99% of all cases.
And just in case someone will use special characters (e.g. slash or hash ...),
the action is likely to fail. The user is able to fix this by himself, by changing the name.

Thinking all over that, I tend to remove the encoding now.
What do you think?